explorer/src/components/Domain/index.tsx

```diff
@@ -0,0 +1,95 @@
+'use client'
+
+import { BlockIcon, DocIcon } from '@/components/icons'
+import { Routes } from 'constants/routes'
+import useDomains from 'hooks/useDomains'
+import Link from 'next/link'
+import { FC, useMemo } from 'react'
+
+export const DomainPage: FC = () => {
+  const { selectedChain } = useDomains()
+
+  const listOfCards = useMemo(
+    () => [
+      {
+        title: 'Nova',
+        description: 'EVM domain',
+        href: `/${selectedChain.urls.page}/${Routes.nova}`,
+        icon: <BlockIcon />,
+        darkBgClass:
+          'dark:bg-gradient-to-b dark:from-purpleLighterAccent dark:via-purpleMedium dark:to-purplePale',
+      },
+      {
+        title: 'Auto-ID',
+        description: 'Identity domain',
+        href: `/${selectedChain.urls.page}/${Routes.autoid}`,
+        icon: <DocIcon />,
+        darkBgClass: 'dark:bg-gradient-to-b dark:from-purpleDeep dark:to-purplePastel',
+      },
+    ],
+    [selectedChain.urls.page],
+  )
+
+  return (
+    <div className='flex w-full flex-col items-center space-y-4'>
+      <div className='w-full max-w-4xl'>
+        <div className='mb-4 w-full rounded-[20px] border border-slate-100 bg-white px-3 py-4 shadow dark:border-none dark:bg-gradient-to-r dark:from-gradientTwilight dark:via-gradientDusk dark:to-gradientSunset sm:p-6'>
+          <div className='mb-10 flex flex-col items-center justify-center'>
+            <h1 className='mb-8 mt-6 text-center text-4xl font-bold text-gray-900 dark:text-white'>
+              Domain
```

Comment:
I would make this plural.
```suggestion
              Domains
```